### PR TITLE
Allow import and export xprv

### LINKED ISSUE #1508: Export private key (xprv)

### LINKED ISSUE #1509: Implement import xprv flow

### PR DESCRIPTION
**Pull request by gabaldon**
close #1508 
close #1509

### CODE REVIEW

File: src/components/steps/ImportXprv.vue [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    },
+    fileInfo(info) {
+      if (info && this.uploadFileError) {
+        this.clearError()
+      }
+      if (info) {
+        this.disabledNextButton = false
+      } else {
+        this.disabledNextButton = true
+      }
```
Comment (Tommytrg) [inline src/components/steps/ImportXprv.vue:85]:
```suggestion
        this.disabledNextButton = !info
```

File: src/components/steps/ImportXprv.vue [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      if (this.xprvError) {
+        this.clearError({ error: this.xprvError.name })
+      }
+    },
+    uploadFileError(error) {
+      if (error) {
+        this.disabledNextButton = true
+      } else {
+        this.disabledNextButton = false
+      }
```
Comment (Tommytrg) [inline src/components/steps/ImportXprv.vue:75]:
```suggestion
        this.disabledNextButton = !!error

```

File: src/components/steps/WalletSeedTypeSelection.vue [hunk 3]
```diff
@@ -24,6 +24,15 @@
           Import a wallet from mnemonics
         </el-button>
       </li>
+      <li class="option">
```
Comment (Tommytrg) [inline src/components/steps/WalletSeedTypeSelection.vue:27]: Could you use only oner <li> with an `v-for`

File: tests/unit/src/components/FileUploader.spec.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
+import { shallowMount, mount } from '@vue/test-utils'
+import FileUploader from '@/components/FileUploader.vue'
+// eslint-disable-next-line import/no-relative-parent-imports
+import claimingFileSuccess from '../../../../TemplateExample'
```
Comment (Tommytrg) [inline tests/unit/src/components/FileUploader.spec.js:4]: Use @ instead of relative paths

**Review by clbartoli — APPROVED** [review]

**Pull request merged**